starter-code/src/App.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import "./App.css";
+
+const links = [
+    { title: "Déclarative", description: "React make it painless to create interartive UI", img: "/images/icon1.png" },
+    { title: "Components", description: "Build encapsulated components that manage their state", img: "/images/icon2.png" },
+    { title: "Single-way", description: "A set of immutable values are passed to the component's", img: "/images/icon3.png" },
+    { title: "JSX", description: "Statically-typed, designed to run on modern browsers", img: "/images/icon4.png" }
+];
+
+
+class App extends Component {
+    render() {
+        return (
+            <div className="section">
+                <div className="header">
+                    <nav className="nav">
```

Comment:
you didn't need to add the class nav- you could have just used the tag nav to add styling